### PR TITLE
Halide tweaks.

### PR DESCRIPTION
**Pull request by vee-effekt**
Okay, I actually don't know how to revert a commit on Git, so this is what I came up with—sorry!

### CODE REVIEW

File: tests/recipes/halide.rs [hunk 1]
```diff
@@ -24,23 +21,19 @@ pub fn halide_rules() -> Ruleset<Pred> {
...
-    // Rational rules up to size 5:
     let rat_only = recursive_rules(
         Metric::Atoms,
         5,
         Lang::new(
             &["-1", "0", "1"],
             &["a", "b", "c"],
             &["-"],
-            &["+", "-", "*", "/", "min", "max"],
+            &["+", "-", "*", "min", "max"],
```
Comment (ajpal) [inline tests/recipes/halide.rs:31]: no more div? why? if it just blows things up too much then maybe we should have it here but not in the `all` workload? it's hard to imagine that learning simple arithmetic rules with div could be *worse*?

Comment (vee-effekt) [reply tests/recipes/halide.rs:31]: it's not worse, but it gets you nothing and adds like 300 rules

Comment (vee-effekt) [reply tests/recipes/halide.rs:31]: difference between 910 and 1232 rules. I feel like 910 looks better if proving power is the same

Comment (ajpal) [reply tests/recipes/halide.rs:31]: whoa that's a big difference. ok leave it out.

**Review by ajpal — APPROVED** [review]

**Pull request merged**